docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -418,6 +418,106 @@ Ansible can also be installed inside a new or existing ``virtualenv``::
     $ source ansible/bin/activate   # Activate the virtual environment
     $ python -m pip install ansible
 
+.. _with_docker:
+
+Installing Ansible with Docker
+------------------------------
+
+This installation method simply uses ansible inside a docker container with ``docker run --rm -it -v <current-location>:/ansible willhallonline/ansible:2.9-alpine ansible``. To reduce the amount of text to write, simply create an alias.
```

Comment:
I don't see us including documentation to a docker container outside of our control.

At the moment, the closest thing to an official container is the ansible-runner container: https://ansible-runner.readthedocs.io/en/latest/intro.html#container-based-execution